TESTS.md

```diff
@@ -4,16 +4,16 @@
 
 If you are planning to run the full suite of tests or work on policy sets or registry modules, you'll need to set up repositories for them in GitHub.
 
-Your policy set repository will need the following: 
+Your policy set repository will need the following:
 1. A policy set stored in a subdirectory `policy-sets/foo`
 1. A branch other than master named `policies`
 
 Your registry module repository will need to be a [valid module](https://www.terraform.io/docs/cloud/registry/publish.html#preparing-a-module-repository).
-It will need the following: 
+It will need the following:
 1. To be named `terraform-<PROVIDER>-<NAME>`
 1. At least one valid SemVer tag in the format `x.y.z`
-[terraform-random-module](ttps://github.com/caseylang/terraform-random-module) is a good example repo. 
-   
+[terraform-random-module](ttps://github.com/caseylang/terraform-random-module) is a good example repo.
```

Comment:
nitpick - since this line is touched anyways - 
```suggestion
[terraform-random-module](https://github.com/caseylang/terraform-random-module) is a good example repo.
```

Reply:
Fixed